src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetRandomCommand.cs

```diff
@@ -276,19 +278,31 @@ private double ConvertToDouble(object o, double defaultIfNull)
         /// List from which random elements are chosen.
         /// </summary>
         [Parameter(ParameterSetName = RandomListItemParameterSet, ValueFromPipeline = true, Position = 0, Mandatory = true)]
+        [Parameter(ParameterSetName = ShuffleSet, ValueFromPipeline = true, Position = 0, Mandatory = true)]
         [System.Management.Automation.AllowNull]
         [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
         public object[] InputObject { get; set; }
 
         /// <summary>
         /// Number of items to output (number of list items or of numbers).
         /// </summary>
-        [Parameter]
+        [Parameter(ParameterSetName = RandomNumberParameterSet)]
+        [Parameter(ParameterSetName = RandomListItemParameterSet)]
         [ValidateRange(1, int.MaxValue)]
         public int Count { get; set; } = 1;
 
         #endregion
 
+        #region Shuffle parameter
+
+        /// <summary>
+        /// If specified, then the command will return all input objects in randomized order.
```

Comment:
You'll need to phrase this with `Gets or sets whether the command (...)` in order to satisfy CodeFactor. 

At some point we'll have to go back through and align the other parameter descriptions with that format, but no need to go that far for this PR. 🙂